cloudvolume/cloudvolume.py

```diff
@@ -183,7 +186,7 @@ def from_numpy(cls, arr, vol_path='file:///tmp/image/'+generate_random_string(),
       raise NotImplementedError
 
     info = cls.create_new_info(num_channels, layer_type, arr.dtype.name, 'raw', resolution, 
-                               voxel_offset, arr.shape[:3], chunk_size=chunk_size, mip_num=mip_num)
+                               voxel_offset, arr.shape[:3], chunk_size=chunk_size, max_mip=max_mip)
     vol = CloudVolume(vol_path, info=info, bounded=True, autocrop=False) 
```

Comment:
Autocrop is False by default as is bounded = True. Why overspecify?

Reply:
deleted the autocrop option.